src/AppInstallerRepositoryCore/Microsoft/SQLiteIndexSource.cpp

```diff
@@ -31,375 +31,894 @@ namespace AppInstaller::Repository::Microsoft
             std::weak_ptr<SQLiteIndexSource> m_source;
         };
 
-        // The IPackageVersion impl for SQLiteIndexSource.
-        struct PackageVersion : public SourceReference, public IPackageVersion
+        namespace V1

```

Comment:
I'm sure this is easier, but it might be worthy to have them in separate files for maintainability.